client/apps/edit/components/content/sections/image_collection/index.coffee

```diff
@@ -61,86 +64,120 @@ module.exports = React.createClass
   removeItem: (item) -> =>
     @setState imagesLoaded: false
     newImages = _.without @props.section.get('images'), item
-    @props.section.set images: newImages
+    @props.section.set(images: newImages)
     @onChange()
 
   onDragEnd: (images) ->
     @setState imagesLoaded: false
-    @props.section.set images: images
+    @props.section.set(images: images)
     @onChange()
 
-  largeImagesetClass: ->
-    imagesetClass = ''
-    if @props.section.get('type') is 'image_set'
-      if @props.section.get('images').length > 3
-        imagesetClass = ' imageset-block'
-      if @props.section.get('images').length > 6
-        imagesetClass = ' imageset-block imageset-block--long'
-    imagesetClass
+  isImageSetWrapping: ->
+    return true if @props.section.get('type') is 'image_set' &&
+     @props.section.get('images').length > 3
```

Comment:
Nitpick: this would be much easier to read with 

```
   if @props.section.get('type') is 'image_set' && @props.section.get('images').length > 3
      return true
```